Modules/Dashboard/Http/Controllers/DashboardWebController.php

```
@@ -0,0 +1,114 @@
+    /**
+     * Display metrics dashboard
+     */
+    public function metrics(Request $request)
+    {
+        return view('dashboard::metrics');
+    }
+
+    /**
+     * View a specific report
+     */
+    public function viewReport(Request $request, int $id)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)